new_jersey/ver2/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <title>New Jersey</title>
+</head>
+<body>
+<section class="container">
+    <section class="widget">
+        <div class="header">
```

Comment:
I suggest using BEM naming methodology (https://ru.bem.info/). Because the header, main, forecast, date, and footer are all elements of the widget block, we should name them like so:

`<div class="widget">`
`<div class="widget__header"></div>`
`</div>`

The same principle shall be applied to other parts of the code as well, such as elements inside the header and footer blocks, etc.

Reply:
When I renamed classes in accordance with the BEM methodology, I encountered the following situation: `.widget` is a block, and the rest of it, according to the methodology, is elements; but in my implementation there are elements that can also be considered as a block, but in fact they are elements (in `.widget` -> `.widget__footer` -> `.widget__additional-info`). `.widget__additional-info` - block for elements `.additional-info__item`, but relative `.widget__footer` is still an element. 
I looked at the presence of so-called "auxiliary blocks", but did not find the implementation of naming without splitting into additional styles. The naming option that I implemented is good?

Reply:
When I renamed classes in accordance with the BEM methodology, I faced the following situation: `.widget` is a block, and the rest of it, according to the methodology, is elements; but in my implementation there are elements that can also be considered as a block, but in fact they are elements (in `.widget` -> `.widget__footer` -> `.widget__additional-info`). `.widget__additional-info` - block for elements `.additional-info__item`, but relative `.widget__footer` is still an element.
I looked at the presence of so-called "auxiliary blocks", but did not find the implementation of naming without splitting into additional styles. The naming option that I implemented can be applied?

Reply:
The approach you used is totally viable. For small blocks, we may use only one block as a "base", and name everything inside it like "block__element". But, in the case of this weather widget, its structure is quite complex. So, using the "1st level" elements as "2nd level" blocks and so on is justified. Good job!